server/index.js

```diff
@@ -28,6 +34,14 @@ app.set('port', port);
 
 app.listen(port, () => {
   console.log(`App listening on port ${app.get('port')}`);
+
+  // add worker if on production environment
+  if (process.env.NODE_ENV === 'production') {
+    setInterval(async () => {
+      worker.init();
```

Comment:
What if the `worker.exec()` starts executing when `worker.init()` has not yet completed. You might like to use `await`

Reply:
@nzediegwu1 `worker.init()` is a synchronous operation, so it's unnecessary to `await`

Reply:
I'll remove the `async` keyword from the setInterval callback function, cos it's unnecessary